### ISSUE TITLE
npm run site error

### ISSUE THREAD

**Issue by kanitw** [vega/vega-lite#1427]
Not sure why I get these exceptions

```
 bundle exec jekyll build --incremental -q && scripts/generate-toc

  Liquid Exception: wrong argument type nil (expected Regexp) in _layouts/page_submenu.html
jekyll 3.1.6 | Error:  wrong argument type nil (expected Regexp)

npm ERR! Darwin 15.3.0
npm ERR! argv "/usr/local/Cellar/node/5.0.0/bin/node" "/usr/local/bin/npm" "run" "build:toc"
npm ERR! node v5.0.0
npm ERR! npm  v3.3.9
npm ERR! code ELIFECYCLE
npm ERR! vega-lite@1.0.11 build:toc: `bundle exec jekyll build --incremental -q && scripts/generate-toc`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the vega-lite@1.0.11 build:toc script 'bundle exec jekyll build --incremental -q && scripts/generate-toc'.
npm ERR! This is most likely a problem with the vega-lite package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     bundle exec jekyll build --incremental -q && scripts/generate-toc
npm ERR! You can get their info via:
npm ERR!     npm owner ls vega-lite
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/kanitw/Documents/_code/_idl/_visrec/vega-lite/npm-debug.log

npm ERR! Darwin 15.3.0
npm ERR! argv "/usr/local/Cellar/node/5.0.0/bin/node" "/usr/local/bin/npm" "run" "site"
npm ERR! node v5.0.0
npm ERR! npm  v3.3.9
npm ERR! code ELIFECYCLE
npm ERR! vega-lite@1.0.11 presite: `tsc && npm run build && bower install && npm run data && npm run build:toc`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the vega-lite@1.0.11 presite script 'tsc && npm run build && bower install && npm run data && npm run build:toc'.
npm ERR! This is most likely a problem with the vega-lite package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     tsc && npm run build && bower install && npm run data && npm run build:toc
npm ERR! You can get their info via:
npm ERR!     npm owner ls vega-lite
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/kanitw/Documents/_code/_idl/_visrec/vega-lite/npm-debug.log
```

@willium   Can you try if you also get this problem?

**Comment by domoritz** [comment]
The problem is `remove:page.base` in _layouts/page_submenu.html

**Comment by domoritz** [comment]
Not sure what's wrong but it seems like jekyll does not propagate the yaml from matter from `usage.html` to `page_submenu.html` any more. Seems like a bug in jekyll.

**Comment by domoritz** [comment]
To debug this, you can remove everything from page_submenu and replace it with `{{ page }}`. Oddly enough you see that the variable doesn't have anything from the yaml front matter in usage.html any more.

**Comment by domoritz** [comment]
The problem was jekyll changing how it handles yaml front matter in 3.0 but apparently it was not broken right away https://github.com/jekyll/jekyll/issues/4246

**Comment by domoritz** [comment]
Fixed in 0e779900a914421d7dbc4fe71326893f57741cd0
